src/modules/Boards/components/AddCard.tsx

```diff
@@ -0,0 +1,31 @@
+import { CardActionArea } from '@mui/material';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+
+export function AddCard(): JSX.Element {
+  return (
+    <Card sx={{ maxWidth: 345 }} style={{ backgroundColor: '#fefefe' }}>
```

Comment:
Both props used for setting styles ⇾ use one of them. I suggest move color to a constant. I'm sure that this color will be used in other part of the project